### ISSUE TITLE
Move OAF, AAFS, code to it's own repository

### ISSUE THREAD

**Issue by pmpowers-usgs** [opensha/opensha-dev#13]
If @mbarall's work is to be the framework behind a USGS sponsored/published product, it will ultimately need to go through review, which will be painful at the 11th hour unless steps are taken to organize things in a way that more developers are coming to expect. There are different ways to do this depending on how any new code is divided between an OpenSHA dependency and an OAF project. It may be that it makes sense just to have the service architecture in a separate project. If this is the case, consider putting service architecture under `gov.usgs.earthquake.oaf.www`.

In this way, the project will get dedicated issue tracking, wiki, dedicated build script etc.

**Comment by mbarall** [comment]
I agree that this needs to happen.  Let me point out that nvanderelst's work on ETAS is also going to become part of this product eventually.

Not only does OAF/AAFS need to go through review, but there also needs to be an ongoing change control process so that changes get reviewed before they go live.

At present, the project depends on the rest of OpenSHA and can't be built separately from OpenSHA.  If it's going to stay that way, then it probably makes sense for it to become opensha-oaf.  This might be a problem for review and change control, since it isn't reasonable to expect a reviewer to look at the rest of OpenSHA, and OpenSHA is constantly changing.

The alternative of moving it all to gov.usgs.earthquake, I think, implies making the code independent of OpenSHA.  This could be painful, particularly for the GUI.

So at this point I don't know what approach is best.
